.github/workflows/pv.yml

```
@@ -0,0 +1,89 @@
+name: Reusable PV workflow
+
+permissions:
+  contents: none
+
+on:
+  workflow_call:
+    inputs:
+      kokkos_version:
+        description: 'The Kokkos Core version to build'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  PR_PV_ONEAPI202410_SYCL:
```

```suggestion
  PR_PV_ONEAPI_2024_2_1_0_SYCL:
```
or at least include the second number in 2024.2.1.0.